ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/Shrinking.hs

```diff
@@ -37,20 +37,30 @@ shrinkPeerSchedules ::
   GenesisTestFull TestBlock ->
   StateView TestBlock ->
   [GenesisTestFull TestBlock]
-shrinkPeerSchedules genesisTest _stateView =
-  let trimmedBlockTree sch = trimBlockTree' sch (gtBlockTree genesisTest)
+shrinkPeerSchedules genesisTest@GenesisTest{gtBlockTree, gtSchedule} _stateView =
+  let PointSchedule {psSchedule} = gtSchedule
+      simulationDuration = duration gtSchedule
+      trimmedBlockTree sch = trimBlockTree' sch gtBlockTree
       shrunkAdversarialPeers =
-        shrinkAdversarialPeers shrinkAdversarialPeer (psSchedule $ gtSchedule genesisTest)
+        shrinkAdversarialPeers shrinkAdversarialPeer psSchedule
           <&> \shrunkSchedule ->
             genesisTest
-              { gtSchedule = mkPointSchedule shrunkSchedule,
-                gtBlockTree = trimmedBlockTree $ mkPointSchedule shrunkSchedule
+              { gtSchedule = PointSchedule
+                  { psSchedule = shrunkSchedule
+                  , psMinDuration = simulationDuration
+                  }
+              , gtBlockTree = trimmedBlockTree shrunkSchedule
               }
       shrunkHonestPeers =
         shrinkHonestPeers
-          (gtSchedule genesisTest)
-          -- No need to update the tree here, shrinking the honest peers never discards blocks
-          <&> \shrunkSchedule -> genesisTest {gtSchedule = shrunkSchedule}
+          psSchedule
+        -- No need to update the tree here, shrinking the honest peers never discards blocks
+        <&> \shrunkSchedule -> genesisTest
+          { gtSchedule = PointSchedule
+            { psSchedule = shrunkSchedule
+            , psMinDuration = simulationDuration
+            }
+          }
    in shrunkAdversarialPeers ++ shrunkHonestPeers
```

Comment:
Is the comment of this function outdated? It is saying
`This does not affect the honest peer`, which is suggesting that there is only one honest peer, and that it isn't shrunk.